src/OpenTelemetry.Collector.AspNetCore/TracerBuilderExtensions.cs

```diff
@@ -60,7 +60,7 @@ public static TracerBuilder AddRequestCollector(this TracerBuilder builder, Acti
             var options = new AspNetCoreCollectorOptions();
             configure(options);
 
-            return builder.AddCollector(t => new AspNetCoreCollector(t, options));
+            return builder.AddAdapter(t => new AspNetCoreCollector(t, options));
```

Comment:
It would make sense to rename all other 'collectors' to adapter like: `AspNetCoreCollector` -> `AspNetCoreAdapter`, `AzureClientsCollector`, ...

Reply:
This makes a lot of sense, although it would mean a change in NuGet package name. There will be no issues in the NuGet packages being renamed?

Reply:
It sounds like that's the way to go, considering #335 states that these are not collectors but else.

IMHO we could rename this, [mark the old packages as deprecated](https://devblogs.microsoft.com/nuget/deprecating-packages-on-nuget-org/) and point to the new ones.

Let's get a confirmation from @lmolkova and/or @SergeyKanzhelev first though.

Reply:
sure, feel free to rename nuget packages. I don't think we need to worry about deprecation (this is still alpha!)

Reply:
I have rename all projects and class now